src/browser/AxePage.ts

```diff
@@ -17,14 +17,37 @@ export function analyze(
   page: Page,
   disabledRules: string[] = [],
   runOptions: RunOptions = {},
+  config?: Spec,
 ): Promise<AxeResults> {
-  return page.evaluate(runAxe, getRunOptions(runOptions, disabledRules));
+  return page.evaluate(runAxe, {
+    options: getRunOptions(runOptions, disabledRules),
+    config,
+  });
 }
 
-function runAxe(options: RunOptions): Promise<AxeResults> {
+function runAxe({
+  options,
+  config,
+}: {
+  options: RunOptions;
+  config?: Spec;
+}): Promise<AxeResults> {
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore This function executes in a browser context.
-  return window.axeQueue.add(() => window.axe.run(document, options));
+  return window.axeQueue.add(() => {
+    if (config) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore This function executes in a browser context.
+      window.axe.configure(config);
+    } else {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore This function executes in a browser context.
+      window.axe.reset();
```

Comment:
Need to `reset` so that one story doesn't affect the others. This is captured in the integration test.